cadcdata/cadcdata/storageinv.py

```diff
@@ -720,7 +721,9 @@ def _get_uris(self, target):
             os.path.dirname(self._data_client.caps.caps_file),
             '.data_uri_scheme_map')
         scheme_url = self._data_client.caps.caps_urls[DATA_RESOURCE_ID].replace('/capabilities', '/uri-scheme-map')
-        content = self._data_client.caps._get_content(scheme_file, scheme_url)
+        content = util.get_url_content(url=scheme_url,
+                                       cache_file=scheme_file,
+                                       refresh_interval=24 * 60 * 60)
```

Comment:
Can this be made into a constant, so that it would be easier to override for testing? I've seen  `CACHE_REFRESH_INTERVAL` and `VERSION_REFRESH_INTERVAL`.

Reply:
It's only used once internally so I don't think turning it into a const brings much value.